### DISCUSSION TITLE
How to deploy wordpress code on static webapp through azure devops

### DISCUSSION THREAD

**Discussion by gouthamkamre68** [Azure/static-web-apps#1264] [Q&A]
HI,

I have WordPress code which contains PHP and HTML and wp-admin etc pushed to azure devops repo but don't have idea how to deploy code to the Azure static web app.

**Comment by superhp** [comment]
Azure SWA doesn't currently support PHP (https://learn.microsoft.com/en-us/azure/static-web-apps/languages-runtimes). So, you can't deploy Wordpress there. But you could separate Wordpress as a backend-only (headless) deployed to WebApp providing data through its API and SWA as a frontend calling that API and rendering with Javascript (https://www.hostinger.com/tutorials/headless-wordpress).

**Reply by gouthamkamre68** [reply]
Thank for your response. Let me try.
